src/spikeinterface/comparison/paircomparisons.py

```diff
@@ -540,6 +563,12 @@ def get_false_positive_units(self, redundant_score=None):
     def count_false_positive_units(self, redundant_score=None):
         """
         See get_false_positive_units().
+
+        Parameters
+        ----------
+        redundant_score : float, default: None
```

Comment:
```suggestion
        redundant_score : float | None, default: None
```

We still need to discuss this. Since this isn't fully settled. So this comment is optional.